### DISCUSSION TITLE
local XCode file are not appearing on the changes and unable to commit

### DISCUSSION THREAD

**Discussion by AruneelDasArcadix** [community/community#52797] [New to GitHub]
Hi,

I have a local project in XCode, which I am trying to update on my github repository. But I am unable to do so.

The local  file that I have are not appearing on the changes and I am unable to commit and push them.

Please help or git me on the process which I need to follow.

Thanks in Advance :)

**Comment by NJacksonDev** [comment]
Below are some steps you can try and hopefully they can help with fixing your issue.

- Make sure that you have initialized the local directory as a Git repository. If you haven't done so already, you can open a Terminal window, navigate to your project directory, and run the command "git init" to initialize the repository.

- Make sure that you have added the files to the Git staging area. You can do this in Xcode by clicking on the source control icon in the toolbar (it looks like a circle with an arrow pointing up) and selecting "Stage All Changes".

- Make sure that you have committed the changes. You can do this in Xcode by clicking on the source control icon in the toolbar and selecting "Commit". In the commit dialog, enter a commit message and click on the "Commit" button.

- Check the Git command line for error messages. You can open a Terminal window, navigate to your project directory, and run Git commands to check the status, commit history, and other information about your local repository.

**Comment by xDhii** [comment]
Did you solve? I'm facing the same issue.
I've been creating some files via XCode, but I'm not able to stage them.
Files aren't being shown when I run "git status" command on terminal.
